pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/Fixtures.java

```diff
@@ -35,8 +33,8 @@ private Fixtures() {
   public static final int MAX_ROWS_IN_SEGMENT = 250000;
   public static final long MAX_TIME_FOR_SEGMENT_CLOSE_MS = 64368000L;
   public static final String TOPIC_NAME = "someTopic";
-  public static final String CONSUMER_FACTORY_CLASS = FakeStreamConsumerFactory.class.getName();
-  public static final String MESSAGE_DECODER_CLASS = FakeStreamMessageDecoder.class.getName();
+  public static final String CONSUMER_FACTORY_CLASS = "some.consumer.factory.class";
+  public static final String MESSAGE_DECODER_CLASS = "some.message.decoder.class";
```

Comment:
why does this need to change?

Reply:
This is because I moved the classes from `pinot-core` to `pinot-segment-local`, and the TransformPipeline does not care about those two properties.
Somehow it turns out these two are used by the tests in `pinot-core` too.
Now I have changed the Fixtures to take two class names, so it can be customized properly